### PR TITLE
Projects page

### LINKED ISSUE #5: Projects page

### PR DESCRIPTION
**Pull request by Ullaakut**
## Goal of this PR

Fixes #5 

This PR also adds a few markdown extensions to make it easier to show an image in the architecture document `nomnoml` example.

### CODE REVIEW

**Review by ricardomgoncalves — APPROVED** [review]
Looks nice

File: content/projects.md [hunk 1]
```diff
@@ -1,21 +1,157 @@
 # Projects
 
+This document describes the organization and workflow for Optakt projects, and acts as a centralized table of contents of project documentation.
```
Comment (Maelkum) [inline content/projects.md:3]: Perhaps use "acts as a centralized table of contents *for* project documentation?"

File: content/projects.md [hunk 2]
```diff
@@ -1,21 +1,157 @@
...
+
 ## Organization
 
-TODO: Explain what is the minimal documentation required for any Optakt project.
-TODO: Explain what how we handle CI and releases with GitHub Actions.
+Any Optakt project should have at least the following:
```
Comment (Maelkum) [inline content/projects.md:7]:
```suggestion
Any Optakt project should at least have the following:
```

File: content/projects.md [hunk 3]
```diff
@@ -1,21 +1,157 @@
...
+
+* Continuous Integration (CI) using [GitHub Actions](https://github.com/features/actions)
+* Automatic Releases when a tag is pushed (using [`goreleaser`](https://github.com/goreleaser/goreleaser) and a GitHub Action)
+* A Pull Request (PR) template that guides contributors into knowing what are the requirements for a successful PR
```
Comment (Maelkum) [inline content/projects.md:11]: Perhaps something like this is more concise.. WDYT?

```suggestion
* A Pull Request (PR) template that guides contributors into the requirements for a successful PR
```

File: content/projects.md [hunk 4]
```diff
@@ -1,21 +1,157 @@
...
+* Proper documentation
+    * License at the root of the repository
+    * README that describes the project, links to Godoc, shows build status and clearly enumerates the project dependencies
+    * Each binary and API within the project should have their own dedicated documentation
```
Comment (Maelkum) [inline content/projects.md:15]: I also use **binary**, but I think **executable** would be a better fit here.

**Review by Maelkum — APPROVED** [review]
Looks great! :)

File: content/projects.md (hunk 2 above)
Comment (Ullaakut) [reply content/projects.md:7]: Both are correct and I personally prefer the form that I used :p

**Pull request merged**